lib/RestrictionsPlugin.js

```diff
@@ -9,7 +9,6 @@
 /** @typedef {import("./Resolver").ResolveStepHook} ResolveStepHook */
 
 const slashCode = "/".charCodeAt(0);
-const backslashCode = "\\".charCodeAt(0);
```

Comment:
Can you describe?

Reply:
We can keep it like this, but I thought about removing it because internally we should handle only posix separator char